mobile/library/common/http/client.cc

```diff
@@ -442,34 +442,34 @@ void Client::DirectStreamCallbacks::latchError() {
   if (info.responseCode().has_value()) {
     error_->error_code_ = Bridge::Utility::errorCodeFromLocalStatus(
         static_cast<Http::Code>(info.responseCode().value()));
-    error_msg_details.push_back(absl::StrCat("RESPONSE_CODE: ", info.responseCode().value()));
```

Comment:
@RenjieTang Are we dealing with details string size limitations that we want to shorten the prefixes?

Reply:
There's a general size limit on the ping data we send to the servers. Strings from all other sources are shortened so we should perform this optimization too.